### PR TITLE
Add Google Cloud as adopter

### PR DESCRIPTION
**Pull request by alculquicondor**
#### What type of PR is this?

/kind documentation

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: site/content/en/docs/adopters/index.md [hunk 1]
```diff
@@ -19,3 +19,6 @@ If you are using Kueue, feel free to open a pull request to add your organizatio
 | [Horizon, Inc.](https://horizon.cc/) | End User | AI training platform | batch/job </br> ... | [@GhangZh](https://github.com/GhangZh) |
 | [FAR AI](https://far.ai/) | End User | AI alignment research nonprofit | batch/job | [@rhaps0dy](https://github.com/rhaps0dy) |
 | [Shopee, Inc.](https://shopee.com/) | End User | Training/batch inference/data processe in AI platform test env | Customized job </br> RayJob </br> ... | [@denkensk](https://github.com/denkensk) |
+| [Google Cloud](https://cloud.google.com/) | Provider | Part of [kit for training ML workloads on TPUs][gcmldemo] | JobSet | [@mrozacki](https://github.com/mrozacki) |
+
+[gcmldemo]: https://cloud.google.com/blog/products/compute/the-worlds-largest-distributed-llm-training-job-on-tpu-v5e
```
Comment (tenzen-y) [inline site/content/en/docs/adopters/index.md:24]: Oh, This is an interesting article.

**Review by tenzen-y — COMMENTED** [review]
/lgtm

**Comment by alculquicondor** [comment]
/hold
I'm verifying with @mrozacki

**Comment by alculquicondor** [comment]
/hold
ready now :)

**Pull request merged**